FWCore/Utilities/test/BuildFile.xml

```diff
@@ -42,8 +42,9 @@
   <use name="cppunit"/>
 </bin>
 
-<bin name="testFWCoreUtilities" file="typeidbase_t.cppunit.cpp,typeid_t.cppunit.cpp,cputimer_t.cppunit.cpp,esinputtag.cppunit.cpp,extensioncord_t.cppunit.cpp,friendlyname_t.cppunit.cpp,signal_t.cppunit.cpp,soatuple_t.cppunit.cpp,transform.cppunit.cpp,callxnowait_t.cppunit.cpp,vecarray.cppunit.cpp,reusableobjectholder_t.cppunit.cpp,propagate_const_t.cppunit.cpp,indexset.cppunit.cpp">
+<bin name="testFWCoreUtilities" file="typeidbase_t.cppunit.cpp,typeid_t.cppunit.cpp,cputimer_t.cppunit.cpp,esinputtag.cppunit.cpp,extensioncord_t.cppunit.cpp,friendlyname_t.cppunit.cpp,signal_t.cppunit.cpp,soatuple_t.cppunit.cpp,transform.cppunit.cpp,callxnowait_t.cppunit.cpp,vecarray.cppunit.cpp,reusableobjectholder_t.cppunit.cpp,propagate_const_t.cppunit.cpp,indexset.cppunit.cpp,fmt_test.cppunit.cpp">
   <use name="cppunit"/>
+  <use name="fmt"/>
```

Comment:
Nothing in `FWcore/Utilities` will have anything to do with `fmt`, so I'm not sure I want a dependence added here (even if it is in the tests). Perhaps `FWCore/MessageService` would be a better fit. @Dr15Jones, what do you think?

Reply:
I agree with @makortel that this does not belong here as we do not want to add unnecessary dependencies. I think it would be best to add to FWCore/MessageLogger since that is where I think we will first make use of it.

Reply:
@Dr15Jones , @makortel , this is just for testing the tool itself. We can add it under Utilities/General/test

Reply:
> @Dr15Jones , @makortel , this is just for testing the tool itself. We can add it under Utilities/General/test

I moved it if we want it in FWCore/MessageLogger I'll move it again

Reply:
Moved in FWCore/MessageService